texastribune/dot
# Bump some dependencies for security updates

#### What's this PR do?

Bumps the versions of
- validator
-  jwks-rsa 
- release-it.

Also various non-breaking updates were patched with a fresh `npm audit`  

#### Why are we doing this? How does it help us?

Gets rid of security vulnerabilities

#### How should this be manually tested?

`npm install` once you're shelled into the app
`npm run build`, then `npm run start` and confirm it builds

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?

No

#### What are the relevant tickets?

https://3.basecamp.com/3098728/buckets/736178/todos/4484062026

#### Have you done the following, if applicable:

**_(optional: add explanation between parentheses)_**

- [ ] Added automated tests? _( )_
- [ ] Tested manually on mobile? _( )_
- [ ] Checked BrowserStack? _( )_
- [ ] Checked for performance implications? _( )_
- [ ] Checked accessibility? _( )_
- [ ] Checked for security implications? _( )_
- [ ] Updated the documentation/wiki? _( )_

#### TODOs / next steps:

- [ ] _your TODO here_
